packages/now-cli/src/commands/domains/rm.ts

```diff
@@ -85,6 +86,18 @@ export default async function rm(
     return 1;
   }
 
+  const projects = await findProjectsForDomain(client, domain.name);
```

Comment:
Running `vc domains rm foo.bar.com` will actually remove `bar.com` + all its relationships. This could be confusing when `vc domains ls` prints all domains+project domains.

We could check that `domain.name === domainName` to ensure we're removing apex domain.